extension/chrome/elements/compose-modules/compose-render-module.ts

```diff
@@ -220,7 +220,7 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
     }
     this.view.recipientsModule.clearRecipients();
     if (option === 'a_forward') {
-      await this.view.quoteModule.addTripleDotQuoteExpandFooterAndQuoteBtn(this.view.replyMsgId, 'forward');
+      await this.view.quoteModule.addTripleDotQuoteExpandFooterAndQuoteBtn(this.view.replyMsgId, 'forward', true);
```

Comment:
I think we should force reload compose box when switching to reply too, as currently when you click `Secure Forward` and then switch to `Secure Reply` message includes attachment, but we doesn't need it for reply.

You can also set `forceReload` to true only when message includes attachment, because we need reload there only for attachments update.

Reply:
OK

Reply:
Please check new change

Reply:
Works well now 👍